tests/apps/tests.py

```diff
@@ -84,25 +90,83 @@ def test_no_such_app(self):
                 pass
 
     def test_no_such_app_config(self):
-        msg = "No module named 'apps.NoSuchConfig'"
+        msg = "Module 'apps' does not contain a 'NoSuchConfig' class."
         with self.assertRaisesMessage(ImportError, msg):
             with self.settings(INSTALLED_APPS=['apps.NoSuchConfig']):
                 pass
 
     def test_no_such_app_config_with_choices(self):
         msg = (
-            "'apps.apps' does not contain a class 'NoSuchConfig'. Choices are: "
-            "'BadConfig', 'MyAdmin', 'MyAuth', 'NoSuchApp', 'PlainAppsConfig', "
-            "'RelabeledAppsConfig'."
+            "Module 'apps.apps' does not contain a 'NoSuchConfig' class. "
+            "Choices are: 'BadConfig', 'MyAdmin', 'MyAuth', 'NoSuchApp', "
+            "'PlainAppsConfig', 'RelabeledAppsConfig'."
         )
-        with self.assertRaisesMessage(ImproperlyConfigured, msg):
+        with self.assertRaisesMessage(ImportError, msg):
             with self.settings(INSTALLED_APPS=['apps.apps.NoSuchConfig']):
                 pass
 
-    def test_default_app_config(self):
-        with self.settings(INSTALLED_APPS=['apps.default_config_app']):
-            config = apps.get_app_config('default_config_app')
-        self.assertIsInstance(config, CustomConfig)
+    def test_no_config_app(self):
+        """Load an app that doesn't provide an AppConfig class."""
+        with self.settings(INSTALLED_APPS=['apps.no_config_app']):
+            config = apps.get_app_config('no_config_app')
+        self.assertEqual(type(config), AppConfig)
```

Comment:
Is there a reason to not use `assertIsInstance()` (like previously) instead of `assertEqual(type(...), ...)`?